### ISSUE TITLE
[1802.06455] Bayesian uncertainty estimation for batch normalized deep networks

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#64]
## 概要
https://arxiv.org/abs/1802.06455
NNのバッチノーマライゼーション(BN)が変分ベイズと等価であること示してBNで予測の分散=不確実性が計算できることを示した論文

## コメント
公式github:
- https://github.com/icml-mcbn/mcbn  # tensorflow

参考サイト:
- https://www.slideshare.net/DeepLearningJP2016/dlbayesian-uncertainty-estimation-for-batch-normalized-deep-networks
